c2rust-analyze/tests/analyze/string_casts.rs

```diff
@@ -10,6 +10,15 @@ pub fn cast_slice_ptr_to_ptr(s: *const [u8]) {
     s as *const u8;
 }
 
+/// For the below disabled (`#[cfg(any())]`ed) tests, they currently crash in the rewriter
+/// due to it not being able to handle implicitly inserted `std::ptr::addr_of!`s yet.
```

Comment:
I would replace this wording with implicitly-inserted `&raw` statements

Reply:
Fixed in 11bf3514929133cd5ee44fe4eb5956bb0e4c07e8.